### PR TITLE
bump package versions, fix lint

### PR DESCRIPTION
**Pull request by MathiasVandePol**

### CODE REVIEW

**Review by ryanwilsonperkin — APPROVED** [review]

File: src/traverse.ts [hunk 1]
```diff
@@ -6,10 +6,6 @@ import {
   simpleTraverse,
 } from '@typescript-eslint/typescript-estree';
 import * as fs from './fs';
-import type {
```
[the remark this answers is not in the document]
Comment (MathiasVandePol) [reply src/traverse.ts:9]: with typescript 5.2.2 you need `module` and `moduleresolution` set to nodenext. This makes it impossible to use `@typescript-eslint/types/dist/generated/ast-spec` as this is not a valid path for the new module resolution.
I tried different ways but was unable to get it it work.
Feel free to give it a try yourself

[the remark this answers is not in the document]
Comment (ryanwilsonperkin) [reply src/traverse.ts:9]: Resolved with an import of the `TSESTree` type namespace

**Comment by ryanwilsonperkin** [comment]
Looks like the new typescript eslint is using syntax that's unsupported by Node v14 that's running in CI. Need to drop support for that if we want to add it in.

File: src/traverse.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (MathiasVandePol) [reply src/traverse.ts:9]: Thanks!

**Comment by smeijer** [comment]
Thank you! Much appreciated! 

FYI, this repo qualifies for [hacktoberfest](https://hacktoberfest.com) contributions if that's your thing.

**Comment by smeijer** [comment]
@all-contributors please add @MathiasVandePol for code

**Pull request merged**